runtime/ast/entitlement_declaration.go

```diff
@@ -162,6 +162,7 @@ type EntitlementMappingDeclaration struct {
 	DocString    string
 	Identifier   Identifier
 	Associations []*EntitlementMapElement
+	Inclusions   []*NominalType
```

Comment:
It might be good to merge `Associations` and `Inclusions`, so that the order of the declarations is preserved. This would be similar to `CompositeDeclaration.Members`, which keeps e.g. fields, functions, etc. in order `Members.desclarations`, but provides getters for the individual kinds of declarations.

This enables the AST to be formatted back into how it was parsed.